### DISCUSSION TITLE
Updating from the command line?

### DISCUSSION THREAD

**Discussion by ianchesal** [NvChad/NvChad#1887] [Q&A] [answered]
In my dotfiles repo [I have an `update` target](https://github.com/ianchesal/dotfiles/blob/main/nvim/nvim.rake#L15
) that keeps all my things fresh. I'm failing to come up with a way to invoke `NvChadUpdate` from the command line such that it doesn't hang waiting for the press of a `q` key on a keyboard after it runs. `--headless` hangs indefinitely.

Is there a command line invocation for NvChadUpdate that'll term after it runs so I can put this all in a Rakefile-driven flow?

**Comment by TravisDart** [comment, marked as answer]
Not sure if this completely answers your question, but I'm using this command to start Neovim, load all plugins, and quit: https://github.com/TravisDart/nvchad-neovim/blob/main/Dockerfile#L25

It works with no issue for me, so you might try something similar for your use case.

**Reply by siduck** [reply]
@TravisDart does it quit properly after everything?

**Reply by TravisDart** [reply]
Yep. That command is used during a docker build, and the build completes successfully without hanging or anything, so that command exists like it should.
